geo/CHANGES.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-*
+* Add `Default` constraint to all `CoordNum`/`CoordFloat` values.
```

Comment:
```suggestion
* BREAKING: Add `Default` constraint to all `CoordNum`/`CoordFloat` values.
```

This would be a breaking change for geo-types right? 

I'm not opposed to the new behavior, but it seems like a pretty small QOL improvement. 

As such, I'd advocate for holding off on merging this until we have a batch of other more significant reasons to break geo-types.

What do other people think?

Reply:
@michaelkirk  I'm ok to hold off the default trait. What about other changes - should i split them into a separate PR?

Reply:
> What about other changes

I'm not sure what remains. Do you mean this?

```diff
-    T: ::num_traits::Float + ::rstar_0_9::RTreeNum,
+    T: CoordFloat + ::rstar_0_9::RTreeNum,
```

Isn't that also a breaking change?

If there's some non-breaking changes that I'be missed that you'd like merged, I'd recommend splitting them out.